src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -598,38 +598,56 @@ void destroyAllOntimeIntervalThreads(std::string tenant)
 
```

Comment:
Compring the test harness that have failed:

```
```
----------- Failing tests ------------------
  o  117_convop_using_standard_ops/postContextEntitiesByEntityId.test
  o  117_convop_using_standard_ops/postContextEntityAttributes.test
  o  117_convop_using_standard_ops/postEntityByIdAttributeByName.test
  o  499_context_providers_basic/query_redirect_operation.test
  o  499_context_providers_basic/query_redirect_three_providers.test
  o  499_context_providers_basic/update_redirect_delete_operation.test
  o  499_context_providers_basic/update_redirect_operation.test
  o  674_servicepath_in_notifications/service_path_in_onchange_notification.test
  o  674_servicepath_in_notifications/service_path_in_ontimeinterval_notification.test
  o  674_servicepath_in_notifications/service_path_in_single_onchange_notification.test
  o  675_servicePath_in_ngsi10_subscriptions/simple_ngsi10_subscription_both_cities_with_isPattern_and_deeper_service_path.test
  o  675_servicePath_in_ngsi10_subscriptions/simple_ngsi10_subscription_exact_city_with_isPattern_and_deeper_service_path.test
  o  675_servicePath_in_ngsi10_subscriptions/simple_ngsi10_subscription_with_isPattern_and_deeper_service_path.test
  o  675_servicePath_in_ngsi10_subscriptions/simple_ngsi10_subscription_with_recursive_service_path.test
  o  675_servicePath_in_ngsi10_subscriptions/simple_ngsi10_subscription_with_recursive_service_path_and_isPattern.test
  o  714_propagate_service_path/no_propagation_of_service_path_when_not_present.test
  o  715_forward_xauth_token_to_cprs/propagate_xauth_token_for_update_and_query_standard_ops.test
  o  716_forward_registered_attributes/forward_attribute_query.test
  o  719_servicepath_for_registrations/forward_attribute_with_servicePath.test
  o  719_servicepath_for_registrations/forward_queries.test
```
```

and the following check of .test containing the "CP" word:

```
fermin@debvm:~/src/fiware-orion/test/functionalTest/cases$ grep -R -c 'CP' * | grep -v '0$' | grep '.test'
117_convop_using_standard_ops/postContextEntitiesByEntityId.test:12
117_convop_using_standard_ops/postEntityByIdAttributeByName.test:12
117_convop_using_standard_ops/postContextEntityAttributes.test:12
499_context_providers_basic/query_redirect_three_providers.test:105
499_context_providers_basic/update_redirect_delete_operation.test:24
499_context_providers_basic/update_redirect_operation.test:46
499_context_providers_basic/query_redirect_operation.test:29
673_xauth_token/auth_token_three_steps.test:12
713_propagate_tenant/propagate_tenant_for_update_and_query_standard_ops.test:34
714_propagate_service_path/propagate_service_path_for_update_and_query_standard_ops.test:38
716_forward_registered_attributes/forward_attribute_query.test:13
716_forward_registered_attributes/forward_attribute.test:16
719_servicepath_for_registrations/register_and_discovery_with_invalid_service_paths.test:3
719_servicepath_for_registrations/register_and_discovery_with_servicepath.test:2
719_servicepath_for_registrations/register_and_discovery_with_many_servicepaths.test:5
719_servicepath_for_registrations/forward_queries.test:45
719_servicepath_for_registrations/forward_attribute_with_servicePath.test:16
```